templates/about/release-cycle.html

```diff
@@ -10,380 +10,542 @@
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="p-tabs__content" tabindex="0" id="editions-and-flavors" role="tabpanel" aria-labelledby="editions-and-flavors-tab">
+          <div class="p-section--shallow">
+            <div class="grid-row">
+              <div class="grid-col-2">
+                <h3 class="p-heading--5">Ubuntu Classic</h3>
+              </div>
+              <div class="grid-col-4">
+                <p>Ubuntu comes in various editions and minimal installs. “Classic” Ubuntu uses deb packages and supports snaps for specific apps, available across different desktop flavors and server/cloud images.</p>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="p-section--shallow">
+            <div class="grid-row">
+              <div class="grid-col-2">
+                <h3 class="p-heading--5">Ubuntu Core</h3>
+              </div>
+              <div class="grid-col-4">
+                <p>Ubuntu Core is an all-snap edition designed for appliances and IoT. With a lightweight, strictly confined architecture, it's highly secure and reliable &ndash; perfect for large-scale, embedded deployments.</p>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted"/>
+          <div class="p-section--shallow">
+            <div class="grid-row">
+              <div class="grid-col-2">
+                <h3 class="p-heading--5">Ubuntu Minimal</h3>
+              </div>
+              <div class="grid-col-4">
+                <p>Ubuntu Minimal is a lightweight image that includes only the essential packages, making it ideal for building custom environments or optimizing deployments for performance and footprint. It provides an efficient foundation for containers, virtual machines, and specialized workloads requiring fine-grained control. Ubuntu Minimal images are available for all major public clouds, including AWS, Google Cloud and Azure.</p>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted"/>
+          <div class="p-section--shallow">
+            <div class="grid-row">
+              <div class="grid-col-2">
+                <h3 class="p-heading--5">Ubuntu Flavors</h3>
+              </div>
+              <div class="grid-col-4">
+                <p>Ubuntu flavors are community-maintained variants that offer different desktop environments and user experiences. The release cycle for flavours may differ from those described here. For release details and support policies specific to a flavor, please consult its official website or documentation.</p>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
There's a cta link here in the copydoc

<img width="630" height="166" alt="Image" src="https://github.com/user-attachments/assets/0dfde66a-621a-446a-8470-ddbf94fb0eda" />